### PR TITLE
Introduce OnboardingSelectFlow screen

### PR DESCRIPTION
**Pull request by JustCaptcha**

### CODE REVIEW

File: universal-login-react/src/playground/App.tsx [hunk 1]
```diff
@@ -100,6 +101,18 @@ export const App = () => {
                     hideModal={() => history.push('/')}
                   />}
               />
+              <Route
+                exact
+                path="/onboardingNew"
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/playground/App.tsx:106]: let's name it `onboardingEmail`

File: universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  domains,
+  actions,
+  placeholder,
+}: WalletSelectorProps) => {
+  const [flow, setFlow] = useState('create');
+
+  const handleConfirmClick = () => {
+    const ensName = 'satoshi93.unilogin.eth';
+    if (flow === 'create') {
+      onCreateClick!(ensName);
```
Comment (xitronix) [inline universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx:22]: why do we have `!` here?

File: universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import {Label} from '../commons/Form/Label';
+
+export const OnboardingSelectFlow = ({
+  onCreateClick,
+  onConnectClick,
+  sdk,
+  domains,
+  actions,
+  placeholder,
+}: WalletSelectorProps) => {
```
Comment (xitronix) [inline universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx:16]: Introduce OnboardingSelectFlowProps

File: universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import React, {useState} from 'react';
+import {useClassFor, classForComponent} from '../utils/classFor';
+import '../styles/base/onboardingSelectFlow.sass';
+import '../styles/themes/UniLogin/onboardingSelectFlowThemeUniLogin.sass';
+import {WalletSelectorProps} from '../WalletSelector/WalletSelector';
+import Input from '../commons/Input';
+import {Label} from '../commons/Form/Label';
+
+export const OnboardingSelectFlow = ({
```
Comment (xitronix) [inline universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx:9]: why does it flow?

Comment (xitronix) [reply universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx:9]: EmailFlowChooser?

File: universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  onCreateClick,
+  onConnectClick,
+  sdk,
+  domains,
+  actions,
+  placeholder,
+}: WalletSelectorProps) => {
+  const [flow, setFlow] = useState('create');
```
Comment (xitronix) [inline universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx:17]: Introduce 
```ts
type EmailFlow = {
   kind: 'create',
   onClick: () => void,
} | {
   kind: 'connect',
   onClick: () => void,
}
}
```

File: universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+          {flow === 'create' && <CreationFlow/>}
+          {flow === 'connect' && <ConnectionFlow/>}
+        </div>
+      </div>
+      <button onClick={handleConfirmClick} className={classForComponent('confirm-btn')}>Confirm</button>
+    </div>
+  );
+}
+
+const CreationFlow = () => {
```
Comment (xitronix) [inline universal-login-react/src/ui/Onboarding/OnboardingSelectFlow.tsx:49]: Why does it Flow?

File: universal-login-react/src/ui/Onboarding/EmailFlowChooser.tsx [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+          {flow.kind === 'create' && <CreationContent/>}
+          {flow.kind === 'connect' && <ConnectionContent/>}
+        </div>
+      </div>
+      <button onClick={flow.onClick} className={classForComponent('confirm-btn')}>Confirm</button>
+    </div>
+  );
+}
+
+const CreationContent = () => {
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/Onboarding/EmailFlowChooser.tsx:45]: This component should have some internal state e.g. ensName or e-mail that will change in the input.

Comment (xitronix) [reply universal-login-react/src/ui/Onboarding/EmailFlowChooser.tsx:45]: The same with ConnectionContent

File: universal-login-react/src/playground/App.tsx [hunk 8]
```diff
@@ -100,6 +101,18 @@ export const App = () => {
                     hideModal={() => history.push('/')}
                   />}
               />
+              <Route
+                exact
+                path="/onboardingEmail"
+                render={({history}) =>
+                  <EmailFlowChooserScreen
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/playground/App.tsx:108]: We are not using most of these props. I think that we should just remove them and add them as soon as we need them.

File: universal-login-react/src/ui/styles/themes/UniLogin/onboardingSelectFlowThemeUniLogin.sass [hunk 9]
```diff
@@ -0,0 +1,45 @@
+.unilogin-component-select-flow.unilogin-theme-unilogin
+  .unilogin-component-user-tabs
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/styles/themes/UniLogin/onboardingSelectFlowThemeUniLogin.sass:2]: I guess we should remove it

**Pull request merged**